lib/recurly/adjustment.php

```diff
@@ -1,5 +1,32 @@
 <?php
-
+/*
+ * class Recurly_Adjustment
+ * @property string $type The type of adjustment to return: charge or credit.
+ * @property string $account The URL of the account for the specified adjustment.
+ * @property string $invoice The URL of the invoice for the specified adjustment.
```

Comment:
Same here I think anything with an `href=""` in the XML will become a `Recurly_Stub` in the object.